scripts/exregional_nonvarcldanl.sh

```diff
@@ -211,12 +228,47 @@ do
   fi
 done
 
+# radar reflectivity on esg grid over each subdomain.
+process_radarref_path=${cycle_dir}/process_radarref${cycle_tag}
+ss=0
+for bigmin in 0; do
+  bigmin=$( printf %2.2i $bigmin )
+  obs_file=${process_radarref_path}/${bigmin}/RefInGSI3D.dat
+  if [ "${IO_LAYOUT_Y}" == "1" ]; then
+    obs_file_check=${obs_file}
+  else
+    obs_file_check=${obs_file}.0000
+  fi
+  ((ss+=1))
+  num=$( printf %2.2i ${ss} )
```

Comment:
So RefInGSI3D.dat_01, RefInGSI3D.dat_02, RefInGSI3D.dat_03 and RefInGSI3D.dat_04 correspond to t+0, t+15, t+30, t+45 sub cycles? Would it be better to use the sub cycle minutes as the suffix instead of 01, 02, 03, 04?

Reply:
Those numbers will be also used in radar tten and the array of radar tten. It is easy to just use the number.